math/src/field/element.rs

```diff
@@ -694,7 +695,7 @@ mod tests {
         fn test_inplace_batch_inverse_returns_inverses(vec in field_vec(10)) {
             let input: Vec<_> = vec.into_iter().filter(|x| x != &FieldElement::<Stark252PrimeField>::zero()).collect();
             let mut inverses = input.clone();
-            FieldElement::inplace_batch_inverse(&mut inverses);
+            let _ = FieldElement::inplace_batch_inverse(&mut inverses);
```

Comment:
Let's `unwrap` here. A `panic` is better than continuing with a inverses that aren't inverses.

Reply:
Ok will do